pkg/profiler/profiler.go

```diff
@@ -118,16 +118,17 @@ type Profile struct {
 
 // TODO(kakkoyun): Unify PID types.
```

Comment:
Should this be removed?

Reply:
Noop. As we discussed above, we need to convert everything to int32.